### ISSUE TITLE
Base Class To-Do List

### ISSUE THREAD

**Issue by Tarkisflux** [SqueeG/awesomeTome#23]
The following classes are either original Tome classes or filling in SRD spaces, and are core classes for TRD and thus high priority for conversion. They need to be updated to use the new class functions, and incorporated into the main document. We can errata them later (and we will want to for some of them)
- <del>Tome Assassin</del> - completed
- Tome Barbarian
- SRD Druid
- Tome Fighter
- Tome Jester
- Tome Knight
- Tome Monk
- SRD Rogue
- Tome Samurai
- Tome Sorcerer - http://www.dnd-wiki.org/wiki/Sorcerer,_Tome_%283.5e_Class%29
- Tome Summoner
- Tome Thief-Acrobat
- SRD Wizard

The following classes are also core TRD but should not be converted yet, pending discussion or more general edits:
- Cid's Bard (pending rework)
- Paladin (pending selection)
- Ranger (pending selection)
- SRD Cleric (potential replacement with wiki Priest)

Secondary priorities include the following, likely in the form of addendums / supplements:
- Anything you particularly want to see added to core, not included on the above list. _Note that it might not actually be core (unless you recompile your own)._
  - Frank's Soldier (I think there was a request for it, but can't find it offhand)
  - Tome Marshal (personal request, I'd like this to be core)
- Fiend / Sphere classes (these will be core if we decide to includes sphere mechanics):
  - True Fiend
  - Fiendish Brute
  - Conduit of the Lower Planes (strong preference for CatharzConduit over original, as errataed here - http://www.dnd-wiki.org/wiki/Conduit_of_the_Lower_Planes_%283.5e_Class%29 )
- A bunch of energy mages:
  - Fire mage
  - Electric mage (Frank's Puppeteer or Kaelik's Stormlord, or both)
  - Ice mage (Maxus's Snowscaper, or Kaelik's Cold Dude, or IGTN's Snowsoul, or Dominicus's Wintersmith, or all of them even)
  - Acid mage
  - Wind mage (IGTN's Windseeker)
- Magic of Blue stuff:
  - Frank's Incarnate (requires completion)
  - Frank's Soulborn
- Koumei's Stuff:
  - Ninja
  - Swashbuckler
  - Witch
  - Warmage
  - Gadgeteer
- Other groups of stuff that I'm forgetting.

If/When we get more contributors we might want to start claiming projects in thread to reduce redundancy, but for now it's probably ok if you just do stuff and commit it.

**Comment by ExplosiveRunes** [comment]
Reformatted and included barbarian to see how long applying the template would take. Took around 10-15 minutes.

**Comment by ExplosiveRunes** [comment]
I started formatting the templar, as it seems likely to be included. The Vows and Styles still need work. Subsections and subsubsections do not seem to be sufficient to divide them. I've included an example of each using section and pushed it, in case someone wants to look at it or work on it before I get back to it. I'll try to fiddle with it tomorrow, but no promises. It's possible it could be done better in some sort of one column table, or maybe temporarily using a twocolumn environment, or something.

**Comment by ExplosiveRunes** [comment]
I've tried something with the optional class features, and updated the templar as a sample. Is this a kind of look that we want to try? I used the mdframed package again, because I knew how to make it work.

@Tarkisflux If it looks right, I think I can make it work without that package if it's still causing compatability problems with you. I just used mdframed because I had some previous code I could quickly adapt to test it out.

**Comment by SqueeG** [comment]
As far as Kantian Paladin is concerned; would IGTN's smite mechanics be something viable, or worthwhile, to bolt on? A few people mentioned that was their favorite part of his build.

**Comment by Tarkisflux** [comment]
@ExplosiveRunes - It looks a bit darker than I might like, and since there was only one example of each I'm not sure if you plan on alternating colors of just having the whole subsubsection be a darker gray than the normal class stuff. I'd also prefer to stick with only 1 type of list, or modify the list spacing to be consistent between them (there's extra space on the avenger styles for some reason). Otherwise I think it looks like a pretty good formatting starting point, and like I like I need someone to help edit the wordcount down a lot (or fluff up other classes for consistency) because that will wind up longer than 3 or 4 other classes combined....

As for the package, when I compiled it myself I got some weird empty white lines between the grey blocks. I'd prefer to either not use that package or borrow you for a bit to see what installed packages you have that might resolve my compile issues.

@SqueeG - I don't think we should worry about it. There were 4 different people asking for a different smite mechanic in thread, and that's not even half of the people who voted for it. Dropping it in as a sidebar might be okay though, it's basically a "remove this stuff, replace with this stuff at these levels" alternate class feature sort of thing anyway.

**Comment by ExplosiveRunes** [comment]
@Tarkisflux The grays are just placeholders (like all the grays in every table, really). On using or not using the mdframed package, it really depends on how easy you want it to be for other people to pick up the usage of that format. The only other way to do the alternating colors (that I've found) is in a tabular type of environment (probably a longtabu in this case). With the mdframed package, it applies the background to an environment that allows you to use commands as you would in paragraph mode (lists, line breaks, whatever). If we use the tabular route, each class would need you to define a table with the proper columns and whatnot to fake it, making it more difficult for someone who wanted to format their own class and submit it.

Classes with big optional features like this are somewhat rare though, so we may not even care. We could just have someone here do it for them.

**Comment by ExplosiveRunes** [comment]
Jeez, it's been a while since I've done... anything on this project. Well, it's been a bit since anyone has done anything on this project. I don't want it to die off, but stuffs been pretty busy lately for me.

That said, in an effort to do something, I formatted the cleric, fighter, and knight.

**Comment by SqueeG** [comment]
I was just thinking the same thing, and I have to apologize. Pretty sure I've said it before but this year has been very hectic and a tad overwhelming at times.

Maybe I could get that out this week. As long as my schedule agrees I will make that my goal. :)

**Comment by Tarkisflux** [comment]
Yeah, I got distracted with life after getting the monster formatting done, and haven't finished reading over the things I'm supposed to read over. It's going to remain hectic for me for another few weeks though, so my stuff will stay half done for a while yet (though I can comment or tweak little things as needed).

But as long as there's motion on this issue, I was wondering about something in general. The Tome classes are pretty sparsely fluffed, and some of the others are much more generously fluffed. Do we care about fluff consistency in the classes? If so should we add it to ones that are missing it or strip it from ones that have it, and should we try to do it at conversion time?

**Comment by SqueeG** [comment]
I guess it depends om what you mean by fluff.

In general I prefer fluff to be as non-specific as possible for classes and especially PrCs. Meaning no "built in" regions or religions or... what ever. It's a lot easier to get certain DMs to approve material when it's not flavor infused.

As for fluff in with class abilities and stuff.
 I like some to be included because it helps me get a feel for a class and what it does/how it reacts/why it does something.

Overly wordy stuff does tend to get skipped, though. XD

**Comment by ExplosiveRunes** [comment]
I'd like each of the entries that are included in the example class template to become standardized in all of the core classes, both in terms of length and tone. The fighter has a 2+ "Playing a fighter section, for example, the tone of which doesn't fit with others.

We also need to choose a "starting age" system, though this is a minor thing it bugs me every time I format a new class and just comment that line out. We could just adopt the simple/moderate/complex system of the wiki if we were so inclined.
